website/docs/reference/resource-configs/database.md

```diff
@@ -45,6 +45,38 @@ This would result in the generated relation being located in the `staging` datab
 </File>
 </TabItem>
 
+<TabItem value="snapshots" label="Snapshots">
+
+<VersionBlock lastVersion="1.8">
+
+Select v1.9 or newer from the version dropdown to view the configs.
```

Comment:
Should we just not include snapshots in the table for versions before 1.9?

Reply:
Unfortunately we can only version the content inside the tab, not the tab itself without replicating the entire setup. When we get that dbt Cloud option in the dropdown we'll have a little more flexibility